### ISSUE TITLE
[Slack] Download All Messages causes crash (0.25)

### ISSUE THREAD

**Issue by cbm64chris** [eul-im/eul#38]
Clicking Download All in a Slack channel causes a crash. I don't see anything obvious at the point of crash but these messages are loaded when the app opens.

```
error	09:48:54.890174 +0100	eul	ERROR: Setting <private> as the first responder for window <private>, but it is in a different window ((null))! This would eventually crash when the view is freed. The first responder will be set to nil.
<private>
default	09:48:55.378445 +0100	eul	UNIX error exception: 17
default	09:48:55.378892 +0100	eul	0x6180000767c0 opened /private/var/db/mds/system/mdsDirectory.db: 50744 bytes
default	09:48:55.380201 +0100	eul	UNIX error exception: 17
default	09:48:55.380385 +0100	eul	UNIX error exception: 17
default	09:48:55.380548 +0100	eul	0x61800007cb40 opened /System/Library/Keychains/SystemRootCertificates.keychain: 385940 bytes
default	09:48:55.386203 +0100	eul	Invalid serial number length (21), must be 1..20
default	09:48:55.393176 +0100	eul	0x610000262f00 opened /Library/Keychains/System.keychain: 81896 bytes
default	09:48:55.397817 +0100	eul	Invalid serial number length (21), must be 1..20
default	09:48:55.403098 +0100	eul	UNIX error exception: 17
default	09:48:55.403420 +0100	eul	UNIX error exception: 17
default	09:48:55.403659 +0100	eul	UNIX error exception: 17
default	09:48:55.403898 +0100	eul	0x610000260ec0 opened /Users/cl011157/Library/Keychains/login.keychain-db: 968956 bytes
default	09:48:55.406968 +0100	eul	Invalid AuthorityKeyIdentifier Extension
default	09:48:55.410026 +0100	eul	Invalid serial number length (21), must be 1..20
```

**Comment by medvednikov** [comment]
Hi @cbm64chris 

Thanks for reporting this. I've received an error report. There's a nil pointer dereference, so a quick thing to fix.

It will be fixed in v0.26, which is going to be live today.

**Comment by voltapp2** [comment]
Hi @cbm64chris

Thanks for reporting this. I've received an error report. There's a nil pointer dereference, so a quick thing to fix.

It will be fixed in v0.26, which is going to be live today.

**Comment by cbm64chris** [comment]
🥇

**Comment by voltapp2** [comment]
Sorry for the delay. This should be fixed in v0.26. Could you please check?

**Comment by cbm64chris** [comment]
Confirmed, downloading messages works correctly.
